### DISCUSSION TITLE
vite + element-plus

### DISCUSSION THREAD

**Discussion by zhaoxiaoniao** [YunYouJun/YunYouJun#9] [Q&A]
我拉了您的vite + element-plus到本地跑起来，element没有加载出来
![CSJ3U(OUO3_B5F{ER~JOKXX](https://user-images.githubusercontent.com/50651602/137834678-de0fb1f2-2fc2-4c60-b1d2-3e4028429bea.png)

**Comment by zhaoxiaoniao** [comment]
包括我自己按照网上npm install element-plus --save 并且修改了main.ts ,也出不来甚至报错了
![1SY`}J0V2 H1WGGY0IGX%TH](https://user-images.githubusercontent.com/50651602/137835271-c14d0ae7-dcfb-4144-9bea-e4a559cdb750.png)

**Comment by YunYouJun** [comment]
请在相关项目进行提问。

[element-plus-vite-starter](https://github.com/element-plus/element-plus-vite-starter) works well.
